src/Utilities.mss

```diff
@@ -234,25 +234,84 @@ function TupletsEqual (t, t2) {
 
 }  //$end
 
-function IsLastNoteInTuplet (bobj) {
+function CountTupletsEndingAtNoteRest(noteRest) {
     //$module(Utilities.mss)
-    next_obj = bobj.NextItem(bobj.VoiceNumber, 'NoteRest');
-
-    if (next_obj = null)
+    tuplet = noteRest.ParentTupletIfAny;
+  
+    if (tuplet = null)
     {
-        return true;
+        return 0;
```

Comment:
What is this return value? I would prefer returning 'false' or 'null' if you mean to not return anything, or to break out of the function.